deploy/cloud/Readme.md

```diff
@@ -0,0 +1,31 @@
+# Sealos cloud cluster image
+
+## usage
+
+### prepare
+
+1. A cloud domain and dns to your k8s cluster. Suppose your domain name is `cloud.example.io`.
+2. A TLS cert that resolves `cloud.example.io` and `*.cloud.example.io`
```

Comment:
How to get or generate it?

Reply:
Provide examples using public DNS and private DNS.

Reply:
May be add docs to links how to issue wildcard cert: https://zerossl.com/features/certificates/
But as install scripts, we cannot help users to auto issue dns-scope cert.